### PR TITLE
Improve DG and UG + Add FAQs

### LINKED ISSUE #175: [PE-D][Tester B] Add new batchmate does not automatically scroll the list view to the bottom
When making use of the add batchmate command, once there are already 7 batchmates in the application, the 8th batchmate added onwards will be added to the bottom of the list view, and the user has to scroll to the bottom to see that the batchmate was added. This could be a suggestion to improve upon :)

For example, calling `add n/charles t/1234567` causes the screen to stay like this, instead of displaying the newly added batchmate.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: alvinjiang1/ped#1

### PR DESCRIPTION
**Pull request by sltsheryl**
Fixes #175

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -348,9 +352,24 @@ _Module prefix refers to the first two characters of every module name._
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install Mass Linkers in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Mass Linkers home folder.
 
+**Q**: Do I need internet connection to use Mass Linkers?<br>
+**A**: No, internet connection is not required. 
+
+**Q**: I am using Mac and I tried opening the `MassLinkers.jar` file by double-clicking it in Finder. It says *"MassLinkers.jar" cannot be opened because it is from an unidentified developer.*<br>
+**A**: Right click the `MassLinkers.jar` file and select `Open with JavaLauncher(default)`.
```
Comment (carriezhengjr) [inline docs/UserGuide.md:359]: Maybe we can say:

Right click the `MassLinkers.jar` file and select `Open`. When a warning message that says _macOS cannot verify the developer of "MassLinkers.jar". Are you sure you want to open it?_ appears, select `Open`. 

Alternatively, you can right click the `MassLinkers.jar` file and select `Open with JavaLauncher(default)`". When the same warning message as the method above appears, select `Open`.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -348,9 +352,24 @@ _Module prefix refers to the first two characters of every module name._
...
+
+**Q**: Will the data of batchmates be saved if the program was not closed via the ```exit``` command?<br>
+**A**: Yes, the data will still be saved. However, we advise using the ```exit``` command for a better user experience.
```
Comment (carriezhengjr) [inline docs/UserGuide.md:362]: Maybe we can be more explicit by saying something like this:

Yes, the data will still be saved if the program is closed by closing the application window directly...

File: docs/UserGuide.md [hunk 3]
```diff
@@ -348,9 +352,24 @@ _Module prefix refers to the first two characters of every module name._
...
+
+**Q**: I have accidentally cleared all the data. Is there any way to undo the changes?<br>
+**A**: Currently, there is no ```undo``` feature and the ```clear``` command is irreversible. Hence, it is extremely important to only use it when you want an empty Student and Module table. However, we are looking into adding ```undo``` and ```redo``` commands for future developments.
+
+**Q**: I have added a batchmate and its row exceeds the current displayed list of batchmates. I have to scroll below to view the newly added batchmate. Is this expected?<br>
```
Comment (carriezhengjr) [inline docs/UserGuide.md:367]: ... exceeds the current displayed list of batchmates __in the Students panel__...

**Review by carriezhengjr — APPROVED** [review]
Some comments you can consider. Otherwise, LGTM.

File: docs/UserGuide.md (hunk 3 above)
Comment (sltsheryl) [reply docs/UserGuide.md]: Made changes to all. Thanks

**Pull request merged**